### PR TITLE
Promote to ONNX commit that has StringNormalizer

### PR DESCRIPTION
**Pull request by yuslepukhin**
Adjust implementation to match ONNX spec.

### CODE REVIEW

**Comment by snnn** [comment]
Hi @yuslepukhin 
Please follow this doc: https://github.com/Microsoft/onnxruntime/blob/master/docs/How_To_Update_ONNX_Dev_Notes.md

**Comment by snnn** [comment]
Please feel free to revise the doc and make it better.  :-)

File: onnxruntime/core/providers/cpu/cpu_execution_provider.cc [hunk 1]
```diff
@@ -515,6 +518,8 @@ void RegisterOnnxOperatorKernels(KernelRegistry& kernel_registry) {
   kernel_registry.Register(BuildKernelCreateInfo<ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 9, int64_t, NonZero)>());
   kernel_registry.Register(BuildKernelCreateInfo<ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 9, string, Where)>());
   kernel_registry.Register(BuildKernelCreateInfo<ONNX_OPERATOR_TYPED_KERNEL_CLASS_NAME(kCpuExecutionProvider, kOnnxDomain, 9, float, Where)>());
+
```
Comment (wschin) [inline onnxruntime/core/providers/cpu/cpu_execution_provider.cc]: Do we need this empty line?

Comment (yuslepukhin) [reply onnxruntime/core/providers/cpu/cpu_execution_provider.cc]: Yes, I need to add a comment // Opset 10

**Comment by snnn** [comment]
The failed Linux test was due to disk space full. I'll solve it tomorrow.

**Comment by skottmckay** [comment]
> [onnx/onnx#1831](https://github.com/onnx/onnx/issues/1831)

Can we update to https://github.com/onnx/onnx/commit/873ddbbc33c6e54d90c5628387edd391fb651dfc instead?

That has some changes to BackendTest, which enables overriding the comparison function. We can also use that to fix the GRU test. 

e.g. code from my branch (need ONNX update to check it in) to do the override to handle the GRU test.

https://github.com/Microsoft/onnxruntime/commit/bb372a923f9d486ade048eb714c48b032a9f1aad

**Comment by yuslepukhin** [comment]
Created https://github.com/onnx/onnx/pull/1851

**Pull request merged**